iati/core/tests/test_default.py

```diff
@@ -47,17 +47,51 @@ def test_default_codelists(self):
         for _, codelist in codelists.items():
             assert isinstance(codelist, iati.core.Codelist)
 
+    def test_default_activity_schemas(self):
+        """Check that the default ActivitySchemas are correct.
+
+        Todo:
+            Check internal values beyond the schemas being the correct type.
+        """
+        schemas = iati.core.default.activity_schemas()
+
+        assert isinstance(schemas, dict)
+        assert len(schemas) == 1
+        for _, schema in schemas.items():
+            assert isinstance(schema, iati.core.ActivitySchema)
+
+    def test_default_organisation_schemas(self):
+        """Check that the default ActivitySchemas are correct.
+
+        Todo:
+            Check internal values beyond the schemas being the correct type.
+        """
+        schemas = iati.core.default.organisation_schemas()
+
+        assert isinstance(schemas, dict)
+        assert len(schemas) == 1
+        for _, schema in schemas.items():
+            assert isinstance(schema, iati.core.OrganisationSchema)
+
     def test_default_schemas(self):
         """Check that the default Schemas are correct.
 
         Todo:
             Check internal values beyond the schemas being the correct type.
-
-            Check for the correct number of Schemas.
         """
         schemas = iati.core.default.schemas()
 
         assert isinstance(schemas, dict)
-        assert len(schemas) == 1
+        assert len(schemas) == 2
         for _, schema in schemas.items():
-            assert isinstance(schema, iati.core.Schema)
+            assert isinstance(schema, (iati.core.ActivitySchema, iati.core.OrganisationSchema))
+
+    @pytest.mark.parametrize("invalid_name", iati.core.tests.utilities.find_parameter_by_type(['str'], False))
+    def test_default_schema(self, invalid_name):
+        """Check that an Error is raised when attempting to load a Schema name that does not exist.
+
+        Type 'str' is excluded since a valid IATI activity name is contained within the fuzzed data.
+
+        """
+        with pytest.raises((ValueError, TypeError)) as excinfo:
```

Comment:
Separating the two types of error into different tests would improve separation of responsibility.